libs/datamodel/core/src/lib.rs

```diff
@@ -224,7 +224,7 @@ fn load_sources(
 pub fn render_datamodel_to_string(datamodel: &dml::Datamodel, configuration: Option<&Configuration>) -> String {
     let mut writable_string = String::with_capacity(datamodel.models.len() * 20);
     render_datamodel_to(&mut writable_string, datamodel, configuration);
-    writable_string
+    reformat(&writable_string, 2).expect("Internal error: failed to reformat introspected schema")
```

Comment:
What if the user wants to use four spaces?

Reply:
It's not configurable in introspection at the moment (AFAIK), so it wouldn't be a regression.

Reply:
Good. Merge away!